### PR TITLE
Create ws parser

### PR DESCRIPTION
**Pull request by kamsi**
https://unctad.atlassian.net/browse/EV2CORE-171

### CODE REVIEW

File: webservice/parser.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+			theirCurrentData = theirCurrentData[segment];
+		});
+		currentResult[lastSegment] = theirCurrentData;
+	});
+
+	return result;
+};
+
+module.exports = function (bp, inputMap, theirData) {
+	var filteredMap = {};
```
Comment (roxanadina) [inline webservice/parser.js]: @kamsi as talked, add as an extra opt the possibility to add 'keypaths' to the output json

File: webservice/parser.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+				mapItemPaths(unStarPath(exports.wsJSON, mapKey),
+					unStarPath(theirData, inputMap[mapKey]), inputMap));
+		} else {
+			filteredMap[mapKey] = inputMap[mapKey];
+		}
+	});
+	return assignValues(filteredMap, exports.wsJSON, theirData);
+};
+
+exports.wsJSON = null;
```
Comment (kamsi) [inline webservice/parser.js]: You're right, it should only be a local variable.

**Comment by mtuchowski** [comment]
@kamsi Resolve conflicts and we can merge this one. It waited long enough…

**Comment by kamsi** [comment]
@mtuchowski done.

**Pull request merged**